spantest/emulator.go

```diff
@@ -30,8 +30,8 @@ import (
 // https://console.cloud.google.com/gcr/images/cloud-spanner-emulator/global/emulator
 const (
 	url     = "gcr.io/cloud-spanner-emulator/emulator"
-	version = "sha256:5d905e581977bd3d543742e74ddb75c0ba65517cf19742089ae1be45b7b8aa49" // 1.5.13
-	image   = url + "@" + version
+	version = "1.5.22"
+	image   = url + ":" + version
```

Comment:
Not a big deal but I like my versions pinned 🥺 

```suggestion
	version = "sha256:ea3a9e70a98bf648952401e964c5403d93e980837acf924288df19e0077ae7fb" // 1.5.22
	image   = url + ":" + version
```

Reply:
Consistency is nice, OK for me 😉